packages/mobile/src/firebase/notifications.ts

```diff
@@ -59,17 +59,20 @@ function* handlePaymentReceived(
     const recipientCache = yield select(recipientCacheSelector)
     const addressToE164Number = yield select(addressToE164NumberSelector)
     const address = transferNotification.sender.toLowerCase()
+    const currency = resolveCurrency(transferNotification.currency)
 
     navigateToPaymentTransferReview(
-      TransactionTypes.RECEIVED,
+      TransactionType.Received,
       new BigNumber(transferNotification.timestamp).toNumber(),
       {
-        value: divideByWei(transferNotification.value),
-        currency: resolveCurrency(transferNotification.currency),
+        amount: {
+          amount: divideByWei(transferNotification.value).toString(),
```

Comment:
Another common source of confusion is whether or not a value is in wei or not. You don't need to fix in this PR but would be cool if we had a wei to type that and avoid potential mistakes.

Reply:
Yep, I will definitely continue thinking about this as improve currencies handling.